Makefile

```diff
@@ -69,7 +69,7 @@ test-fast:
 
 .PHONY: e2e
 e2e:
-	pytest -n auto -v --cov=pyquil test/e2e
+	pytest -n 1 -v --cov=pyquil test/e2e
```

Comment:
It's better to isolate these tests from each other by running synchronously, since they're trying to test out concurrency now